descriptors/ModuleDescriptor-template.json

```diff
@@ -196,11 +196,11 @@
     },
     {
       "id": "instance-storage",
-      "version": "7.1"
+      "version": "7.2"
     },
     {
       "id": "instance-storage-batch",
-      "version": "0.1"
+      "version": "0.4"
```

Comment:
Why has this been moved on 3 versions, is that because it wasn't in previous versions and should've been?

Reply:
In raml file the version is 0.3, so i decided that somebody forgot to change the version here

Reply:
Ok, thanks for spotting this